### PR TITLE
Add unit tests

### LINKED ISSUE #19: Add unit tests
need to add unit tests to project

### PR DESCRIPTION
**Pull request by rstm-sf**
Resolve #19

### CODE REVIEW

File: FastExcel/FastExcel.cs [hunk 1]
```diff
@@ -109,9 +109,9 @@ internal void CheckFiles()
                 }
                 else if (!ExcelFile.Exists)
                 {
-                    string exceptionMessage = string.Format("Input file '{0}' does not exist", ExcelFile.FullName);
+                    var exceptionMessage = $"Input file '{ExcelFile.FullName}' does not exist";
                     ExcelFile = null;
-                    throw new Exception(exceptionMessage);
+                    throw new FileNotFoundException(exceptionMessage);
```
Comment (rstm-sf) [inline FastExcel/FastExcel.cs]: @mrjono1, hello!

Changed the type of exception. This is normal?

Comment (mrjono1) [reply FastExcel/FastExcel.cs]: looks like a good fix

**Comment by mrjono1** [comment]
thanks heaps for this PR, hopefully I'll have some time soon to check out what is going on in appVeyor

**Comment by rstm-sf** [comment]
> MSBuild auto-detection: using msbuild version '15.9.21.664' from 'C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\bin'.
nuget : C:\Program Files\dotnet\sdk\2.2.109\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets(137,5): error NETSDK1045: The current .NET SDK does not support targeting .NET Core 3.0.  Either target 
.NET Core 2.2 or lower, or use a version of the .NET SDK that supports .NET Core 3.0. [C:\projects\fastexcel-43v9o\FastExcel.Tests\FastExcel.Tests.csproj]

Need to change image on Visual Studio 2019

**Comment by mrjono1** [comment]
@rstm-sf I have fixed the build, it also released to nuget... i better do some work to stop that on open PRs

**Comment by rstm-sf** [comment]
@mrjono1 thanks for watching!

Regarding PR, I'm sorry, I had little time for finish. I think that in the near future I will be able to add new tests if I figure out what can be added. Or I just open PR if I can’t figure out what can be added

**Comment by rstm-sf** [comment]
I could not figure out how to reproduce the remaining cases

**Comment by mrjono1** [comment]
Thankyou again ill review this soon

**Pull request merged**